pkg/volume/iscsi/iscsi.go

```diff
@@ -52,8 +52,8 @@ func (plugin *ISCSIPlugin) Name() string {
 	return ISCSIPluginName
 }
 
-func (plugin *ISCSIPlugin) CanSupport(spec *api.Volume) bool {
-	if spec.ISCSI == nil {
+func (plugin *ISCSIPlugin) CanSupport(spec *volume.InternalVolume) bool {
+	if spec.VolumeSource.ISCSI == nil {
```

Comment:
It is not in the PersistentVolumeSource struct.  I figured adding it would be a different PR, as you've suggested in other PRs in the recent past (the NFS pointer made its way into one).